web/js/map/natural-events/util.js

```diff
@@ -9,10 +9,12 @@ import { CRS } from '../../modules/map/constants';
 *
 * @param  {Object} clusterPoint
```

Comment:
This function definition should include "proj"
(I realize this isn't related to your work but wanted to mention it since I noticed it)